src/helpers/eventInstance.ts

```diff
@@ -0,0 +1,69 @@
+import type mongoose from "mongoose";
+import type {
```

Comment:
How can this file be refactored so that functions to handle different intervals can be worked on simultaneously by multiple contributors minimizing merge conflicts?

We'll need to do this to speed up development time. Make a proposal because this approach will need to be done and is stated in the original issue

Reply:
@palisadoes 

Ok we can refactor this like

 **eventInstance/
  ├── index.ts
  ├── daily.ts
  ├── weekly.ts
  └── monthly.ts**

I have completed all the features, and now I only need the pull requests to be reviewed quickly. If so, I believe we can finish all the features in a very short amount of time